src/map/clif.cpp

```diff
@@ -6872,19 +6847,18 @@ void clif_heal(int fd,int type,int val) {
 /// 0148 <id>.L <type>.W
```

Comment:
```suggestion
/// 0148 <id>.L <type>.W (ZC_RESURRECTION)
```
And please remove it above (cant suggest via github...)